ingestion/functions/parsing/common/python/parsing_lib/parsing_lib_test.py

```diff
@@ -1,4 +1,7 @@
-
+# Tests for the parsing library common logic.
+# If you come from a unittest background and wonder at how requests_mock
+# arrives by magic here, check out
```

Comment:
It's :mage: all the way down.

Reply:
yeah this pytest magic is real magic, at least with unittest you setup your mocks before each tests so it's obvious where it comes from.